src/controllers/scripting/controllerscriptenginebase.cpp

```diff
@@ -105,7 +105,7 @@ void ControllerScriptEngineBase::showScriptExceptionDialog(
         errorText = QString("Uncaught exception at line %1 in passed code.")
                             .arg(line);
     } else {
-        errorText = QString("Uncaught exception at line %1 in file %2.")
+        errorText = QString("Uncaught exception at line %1 in file %2:%1 ")
```

Comment:
This is redundant. `%1` is used twice.

Reply:
I know, I wanted the new message to have the proposed format, but I also didn't want to alter it too much from how it looks in the case were there was no associated file. I figured since it was only 4 chars at max being duplicated, it would be fine.

Reply:
How about:

- `somefile.js:123: Uncaught exception`
- `<passed code>:123: Uncaught exception`

Reply:
I'd prefer `Uncaught exception: somefile.js:123: <error message / stack trace / etc>`

Reply:
> Uncaught exception: somefile.js:123: <error message / stack trace / etc>

I like this proposal.